tracker-neuralnet/model_adapters.cpp

```diff
@@ -310,13 +328,18 @@ std::optional<PoseEstimator::Face> PoseEstimator::run(
 
     try
     {
+        std::vector<const char *> input_names, output_names;
+        std::transform(input_names_.begin(), input_names_.end(), std::back_inserter(input_names),
```

Comment:
Don't allocate memory in the main loop.

Reply:
Fixed by adding a vector member for C strings.